<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QFrame" name="frame_18">
     <property name="styleSheet">
      <string notr="true">
border-radius: 15px;
background-color: rgb(212, 212, 255);

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="topMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="login">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Вход&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="loginEdit">
              <property name="styleSheet">
               <string notr="true">height: 50px;
background-color:white;
border-radius:5px;
border: 2px solid gray;
font: 30pt;</string>
              </property>
              <property name="placeholderText">
               <string>Логин</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="passwordEdit">
              <property name="styleSheet">
               <string notr="true">height: 50px;
background-color:white;
border-radius:5px;
border: 2px solid gray;
font: 30pt;</string>
              </property>
              <property name="placeholderText">
               <string>Пароль</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="loginIncorrectText">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">	 font-weight: bold;
	 font-size: 25px;</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;⠀&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Не зарегистрированы?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="registrationButton">
                <property name="styleSheet">
                 <string notr="true"> 	 height: 50px;
	 width: 200px;
 	 box-sizing: border-box;
	 background-color: white;
 	 border-radius: 5px;
	 font-weight: bold;
	 font-size: 25px;
</string>
                </property>
                <property name="text">
                 <string>Регистрация</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="enterButton">
              <property name="styleSheet">
               <string notr="true"> 	 height: 50px;
	 width: 200px;
 	 box-sizing: border-box;
	 background-color: white;
 	 border-radius: 5px;
	 font-weight: bold;
	 font-size: 25px;
</string>
              </property>
              <property name="text">
               <string>Войти</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="registration1">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0,0">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Регистрация&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="loginRegEdit">
              <property name="styleSheet">
               <string notr="true">height: 50px;
background-color:white;
border-radius:5px;
border: 2px solid gray;
font: 30pt;</string>
              </property>
              <property name="placeholderText">
               <string>Логин</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="paswordRegEdit">
              <property name="styleSheet">
               <string notr="true">height: 50px;
background-color:white;
border-radius:5px;
border: 2px solid gray;
font: 30pt;</string>
              </property>
              <property name="placeholderText">
               <string>Пароль</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="regNextButton">
              <property name="styleSheet">
               <string notr="true"> 	 height: 50px;
	 width: 200px;
 	 box-sizing: border-box;
	 background-color: white;
 	 border-radius: 5px;
	 font-weight: bold;
	 font-size: 25px;
</string>
              </property>
              <property name="text">
               <string>Продолжить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="registration2">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Регистрация&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="nameEdit">
              <property name="styleSheet">
               <string notr="true">height: 50px;
background-color:white;
border-radius:5px;
border: 2px solid gray;
font: 30pt;</string>
              </property>
              <property name="placeholderText">
               <string>Имя</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="secondnameEdit">
              <property name="styleSheet">
               <string notr="true">height: 50px;
background-color:white;
border-radius:5px;
border: 2px solid gray;
font: 30pt;</string>
              </property>
              <property name="placeholderText">
               <string>Фамилия</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ageEdit">
              <property name="styleSheet">
               <string notr="true">height: 50px;
background-color:white;
border-radius:5px;
border: 2px solid gray;
font: 30pt;</string>
              </property>
              <property name="placeholderText">
               <string>Возраст</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="heightEdit">
              <property name="styleSheet">
               <string notr="true">height: 50px;
background-color:white;
border-radius:5px;
border: 2px solid gray;
font: 30pt;</string>
              </property>
              <property name="placeholderText">
               <string>Рост</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="weightEdit">
              <property name="styleSheet">
               <string notr="true">height: 50px;
background-color:white;
border-radius:5px;
border: 2px solid gray;
font: 30pt;</string>
              </property>
              <property name="placeholderText">
               <string>Вес</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="genderComboBox">
              <property name="styleSheet">
               <string notr="true">#genderComboBox{
	height: 50px;
	background-color:white;
	border-radius:5px;
	border: 2px solid gray;
	font: 30pt;
}

#genderComboBox::drop-down{
	border: 0px;
}

#genderComboBox::down-arrow{
	image: url(:/icon/img/arrowdown.png);
	width: 12px;
	height:12px;
	margin-right: 15px;	
}

#genderComboBox::on{
	border:4px solid #c2dbfe;
}

#genderComboBox QListView{
	font-size:12px;
	border: 1px solid rgba(0,0,0,10%);
	padding: 5px;
	background-color:#fff;
	outline:0px;
}

#genderComboBox QListView::item {
	paddiong-left: 10px;
	background-color: #fff
}

#genderComboBox QListView::item:hover {
	background-color:#1e90ff;
}

#genderComboBox QListView::item:selected {
	background-color:#1e90ff;
}</string>
              </property>
              <property name="currentText">
               <string>Мужчина</string>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <property name="duplicatesEnabled">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>Мужчина</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Женщина</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="regIncorrectText">
              <property name="styleSheet">
               <string notr="true">	 font-weight: bold;
	 font-size: 25px;</string>
              </property>
              <property name="text">
               <string>⠀</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="regEndButton">
              <property name="styleSheet">
               <string notr="true"> 	 height: 50px;
	 width: 200px;
 	 box-sizing: border-box;
	 background-color: white;
 	 border-radius: 5px;
	 font-weight: bold;
	 font-size: 25px;
</string>
              </property>
              <property name="text">
               <string>Завершить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
